caddytest/integration/stream_test.go

```diff
@@ -358,9 +351,9 @@ func TestH2ToH1ChunkedResponse(t *testing.T) {
 	}
 }
 
-func testH2ToH1ChunkedResponseServeH1(t *testing.T) *http.Server {
+func testH2ToH1ChunkedResponseServeH1(harness *caddytest.TestHarness, t *testing.T) *http.Server {
```

Comment:
Need this be a pointer?

Reply:
yes, in case we add something that can't be passed by value, or modify testharness to be a interface.

 there is no reason to pass the bare struct